modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/snapshot/SnapshotSubcommand.java

```diff
@@ -29,7 +29,10 @@
     CREATE("create"),
 
     /** Sub-command to cancel running snapshot. */
-    CANCEL("cancel");
+    CANCEL("cancel"),
+
+    /** Sub-command to get consistent IDs of the nodes on which the snapshot operation is in progress. */
```

Comment:
or 
Sub-command to checks running snapshot operation is in progress on the nodes?